src/components/auth/profile/UpdateBirthdayModal.tsx

```diff
@@ -16,6 +16,29 @@ import { format, parse, isDate } from 'date-fns'
 import { FormikHelpers } from 'formik'
 import FormTextField from 'components/common/form/FormTextField'
 
+const PREFIX = 'UpdateBirthdayModal'
+
+const classes = {
+  modal: `${PREFIX}-modal`,
+  close: `${PREFIX}-close`,
+}
+
+const StyledModal = styled(Modal)({
+  [`& .${classes.modal}`]: {
+    position: 'absolute',
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 650,
+    backgroundColor: '#EEEEEE',
```

Comment:
Should probably add this color somewhere in the theme

Reply:
We can also use the default MUI colors palette - `gray[200]` 

https://mui.com/material-ui/customization/color/#examples